toolkits/global/packages/global-card/scss/50-components/card.scss

```diff
@@ -103,13 +101,6 @@
 // modifiers
 // ---------
 
-// --flush
```

Comment:
sorry to open the debate on a closed PR. I've realised this class is hugely used across nature pages and removing it will impact the spacing. Do you think I can put it back? @Heydon

Reply:
no worries :)